src/mmw/mmw/urls.py

```diff
@@ -28,9 +29,10 @@
                                namespace='rest_framework')),
     url(r'^admin/', include(admin.site.urls)),
     url(r'^accounts/', include(registration.backends.default.urls)),
-    url(r'^api/bigcz/', include(apps.bigcz.urls)),
-    url(r'^api/geocode/', include(apps.geocode.urls)),
-    url(r'^api/modeling/', include(apps.modeling.urls)),
+    url(r'^mmw/bigcz/', include(apps.bigcz.urls)),
```

Comment:
This will require a corresponding change in `data_catalog/models.js` which is still calling `/api/bigcz/search`.  If you switch to `?bigcz=true`, the search won't work.  

On the topic, perhaps `mmw/bigcz/...` seems a little off.  I'd be open to `mmw, api, bigcz` being the high level non-html rendering routes.  But it's an internal app URL, so not critical.